pkg/translator/jaeger/jaegerproto_to_traces.go

```diff
@@ -274,16 +274,17 @@ func setInternalSpanStatus(attrs pcommon.Map, dest ptrace.SpanStatus) {
 	statusExists := false
 
 	if errorVal, ok := attrs.Get(tracetranslator.TagError); ok {
```

Comment:
I would rather add type check here
```suggestion
	if errorVal, ok := attrs.Get(tracetranslator.TagError); ok && errorVal.Type() == pcommon.ValueTypeBool  {
```